pkg/pipelines/bootstrap.go

```diff
@@ -112,16 +116,35 @@ func Bootstrap(o *BootstrapOptions) error {
 	route := routes.Generate(namespaces["cicd"])
 	outputs = append(outputs, route)
 
-	//  Create Service Account, Role, Role Bindings, and ClusterRole Bindings
-	outputs = append(outputs, createRoleBindings(namespaces)...)
+	// Create Service Account
+	sa := createServiceAccount(meta.NamespacedName(namespaces["cicd"], saName))
+
+	if isInternalRegistry {
```

Comment:
Could probably move this into a separate function that returned the correct set of outputs?

Reply:
Moved